src/components/App.js

```diff
@@ -49,41 +42,83 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
           </div>
         </div>)
     } else {
-      optionBulletForm.push(<RadioButtonOptions key={title} options={options[chosenType][title]} setOptions={setOptions} 
-      selectedOption={info[title]}
-      title={title}/>);
+      optionBulletForm.push(
+      <div key={title} className="option">
+        <div className="option-title">{title}</div>
+        <div className="option-menu">
+        <Dropdown key={title} options={options[chosenType][title]} setOptions={setOptions} 
+        title={title}
+        onChange={(wordTypeForm) => {setOptions(title, wordTypeForm.value)}}
+        value={info[title]}
+        />
+        </div>
+      </div>)
+      ;
     }}
-
+  let languageButton = [];
+  let nativeBackground = '';
+  for(let i = 0; i < languageList.length; i++){
+    let styleName = 'languages';
+    if(selectedLanguage === languageList[i]){
+      nativeBackground = nativeLanguage[languageList[i]];
+      styleName = 'selectedlanguage';
+      languageButton.push(
+        <span>
+          <button
+            key={i}
+            className={styleName}
+            onClick={() => {changeLanguage(languageList[i]);}}>
+            <div className='nativeText'>{nativeLanguage[languageList[i]]}</div>
```

Comment:
if this className is the only difference between the two, you should make it a variable instead so you can avoid repeating the button component (code dup)